views/media/index.tt

```diff
@@ -9,24 +9,134 @@ This program is free software; you may redistribute it and/or modify it under
 the same terms as Perl itself.  For a copy of the license, please reference
 'perldoc perlartistic' or 'perldoc perlgpl'.
 %]
-[% sections.title = '.title' | ml %]
+[% sections.title = '.title2' | ml %]
 
-[% dw.need_res(
+[%- CALL dw.active_resource_group( "foundation" ) -%]
+
+[% dw.need_res( { group => "foundation" }
     "stc/media.css"
 ) %]
 
-<p>[% ".intro" | ml %] <a href="[%site.root%]/file/edit">[% '.intro.edit' | ml %]</a></p>
+[% sections.head = BLOCK %]
+<script>
+function updateEmbed ( evt ) {
+    var thumbnail_value = evt.target.value;
+    var name = evt.target.id.replace(/thumbnail_sizes/, 'embed-thumb');
+    var url = document.getElementById(name).value;
+    document.getElementById(name).value = url.replace(/\d{3,4}x\d{3,4}/, thumbnail_value + 'x' + thumbnail_value);
+}
+
+window.onload = function() {
+  var dropdowns = document.getElementsByClassName('embed-select');
+  for (elem of dropdowns) {
```

Comment:
I'm not sure exactly what this is trying to do, but it's giving me a JS error in Safari:

`TypeError: elem of dropdowns is not a function. (In 'elem of dropdowns', 'elem of dropdowns' is undefined)`